framework/test/unit/modules/dynamic_rewards/module.spec.ts

```diff
@@ -47,6 +47,9 @@ import {
 } from '../../../../src/modules/reward/constants';
 
 describe('DynamicRewardModule', () => {
+	const defaultRoundLength = 103;
+	const defaultNumberOfActiveDelegates = 101;
```

Comment:
What's the convention about read only params constants? Shouldn't we use UPPERCASE? 🤔

Reply:
it's a constant (variable) within a function, so i think it shouldnt be uppercase